hack/common.sh

```diff
@@ -19,6 +19,9 @@ export OCP_VERSION="${OCP_VERSION:-4.11}"
 export OPERATOR_VERSION="${OPERATOR_VERSION:-4.11}"
 export GATEKEEPER_VERSION="${GATEKEEPER_VERSION:-v0.2.0}"
 
+# the metallb-operator deployment and test namespace
+export OO_INSTALL_NAMESPACE="${OO_INSTALL_NAMESPACE:-openshift-metallb-system}"
```

Comment:
why we need this variable?

Reply:
for the metallb-operator tests https://github.com/openshift/metallb-operator/blob/a2f8cb1d9d96a3729c4dd4799aebf1dfc7751cf5/test/e2e/functional/tests/e2e.go#L31